about.html

```diff
@@ -0,0 +1,220 @@
+        >
+          AiAsum Bootcamp '24, also will focus to bring kids interest to
+          computer programming through drag and drop coding.
+        </p>
+
+        <div
+          class="border-2 md:w-[700px] w-full flex justify-center items-center"
+        >
+          <img
+            src="./images/bootcamp.png"
+            alt="bootcamp"
+            class="w-[300px] md:w-[450px] self-center"
+          />
+        </div>
+      </section>
+      <section
+        id="past-event-section"
+        class="w-full py-[60px] flex flex-col items-center gap-[30px]"
+      >
+        <h2
+          class="text-2xl capitalize text-center text-veryDarkBlue font-semibold"
+        >
+          See past events of the AiAsum bootcamp
+        </h2>
+        <p
+          class="w-full md:w-[750px] text-center p-4 text-veryDarkBlue text-xl"
+        >
+          Our Bechem and Accra bootcamps attracted 2000+ students on a
+          transformative coding journey. We empower individuals across Ghana
+          with essential coding skills, creating tech opportunities.
+        </p>
+
+        <div class="grid grid-cols-1 md:grid-cols-2 gap-2">
+          <div
+            class="flex flex-col items-center py-[6rem] bg-lightPink gap-[20px] w-[330px] md:w-[380px] lg:w-[450px] bg-contain bg-blend-multiply bg-[url('../images/SpeedUP-Africa-Ashesi-University.jpg')]"
+            
+          >
+            <p class="text-ash font-bold text-3xl">2021</p>
+            <p class="text-ash font-bold text-xl">
+              AiAsum Bootcamp 2021 in Bechem
+            </p>
+          </div>
+          <div
+            class="flex flex-col items-center py-[6rem] bg-lightPink gap-[20px] w-[330px] md:w-[380px] lg:w-[450px] bg-contain bg-blend-multiply bg-[url('https://africa-school-bucket3.s3.amazonaws.com/static/img/school/2022/11/13/yitgh.png')]"
+            
+          >
+            <p class="text-ash font-bold text-3xl">2023</p>
+            <p class="text-ash font-bold text-xl">
+              AiAsum Bootcamp 2023 in Accra
+            </p>
+          </div>
+        </div>
+      </section>
+      <section
+        id="sponsers-section"
+        class="w-full py-[60px] flex flex-col items-center gap-[30px] bg-veryDarkBlue"
+      >
+        <h2 class="text-2xl capitalize text-ash font-semibold">Sponsors</h2>
+        <div class="flex gap-5 flex-wrap justify-center">
+          <img
+            src="./images/microverse.png"
+            alt="microverse"
+            class="w-[9rem] aspect-auto"
+          />
+          <img
+            src="./images/Google_2015_logo.svg.png"
+            alt="microverse"
+            class="w-[9rem]"
+          />
+          <img src="./images/microsoft.png" alt="microverse" class="w-[9rem]" />
+          <img
+            src="./images/coursera-wordmark-blue.svg"
+            alt="microverse"
+            class="w-[9rem]"
+          />
+        </div>
+      </section>
+    </main>
```

Comment:
- [ ] Every content is awesome but you should stick to the design as much as possible. The pages use the exact same colors, for text, backgrounds, borders, etc, from the original design. I would suggest you please use same font for the content to match the original design. You can use the google font and use lato with Thin 100 Italic. Please check this 
`<style>
@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,700;1,100&display=swap');
</style>`
### Original design
![original](https://github.com/AsumVictor/AiAsum-bootcamp-website/assets/81727988/eae5da88-e1b3-4642-8c07-975d83cb9c54)

### Your design
![his](https://github.com/AsumVictor/AiAsum-bootcamp-website/assets/81727988/6ab5f9db-e9a3-4e1f-a6d6-466d227ec5e0)